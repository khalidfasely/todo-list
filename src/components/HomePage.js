import NewItem from "./NewItem";
import Header from "./Header";

const HomePage = () => {
    return (
        <div>
            <Header />
            <NewItem />
        </div>
    )
};

export default HomePage;