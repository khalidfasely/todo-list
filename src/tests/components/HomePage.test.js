import React from 'react';
import { render } from '@testing-library/react';
import HomePage from '../../components/Header';
jest.mock('../../components/Header', () => () => <div>Header Component</div>);
jest.mock('../../components/NewItem', () => () => <div>New Item Component</div>);
jest.mock('../../components/TodoList', () => () => <div>Todo List Component</div>);


test('Should render HomePage component correctly', () => {
  const { asFragment } = render(<HomePage />);
  expect(asFragment()).toMatchSnapshot();
});