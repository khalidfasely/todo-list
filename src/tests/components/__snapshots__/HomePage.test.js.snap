// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render HomePage component correctly 1`] = `
<DocumentFragment>
  <div>
    Header Component
  </div>
</DocumentFragment>
`;
